lib/syskit/cli/log_runtime_archive.rb

```diff
@@ -196,6 +198,17 @@ def self.transfer_dataset(
                 log_transfer_results(dataset_path, result, logger: logger)
             end
 
+            # Renames the info.yml file
+            def self.rename_info_file(dataset_path)
+                info_file = dataset_path / "info.yml"
+                return unless info_file.file?
+
+                new_info_file =
+                    dataset_path / "info_#{Time.now.strftime('%Y%m%d-%H%M')}.yml"
```

Comment:
Is this from a talk you in person with someone? Apart from the source of the info not being obvious in the file name (which I think is not that big a deal when the info mostly used by other tools), I think you still run the risk of conflict if you stop at the "minutes" in the name. You should at least include the seconds in the name (or ms if you are feeling zealous)

Reply:
I have no idea why she did this ...